torch/_inductor/quantization.py

```diff
@@ -0,0 +1,144 @@
+import torch
```

Comment:
This file should go in the fx_passes folder.

Reply:
OK. It's done.

Reply:
Hi @jansel Please review again. Thanks!